src/r-rig/devcontainer-feature.json

```diff
@@ -14,6 +14,17 @@
 			"default": "release",
 			"description": "Select version of R, if not the latest release version."
 		},
+		"vscodeRSupport": {
+			"type": "string",
+			"enum": [
+				"minimal",
+				"languageserver",
+				"full",
+				"none"
+			],
+			"default": "minimal",
+			"description": "Install R packages to make vscode-R work. languageserver means the `languageserver` package, full means plus the `httpgd` package."
```

Comment:
a bit unclear the difference between minimal and none?  

Also, I have started to set defaults to err on the inclusive rather than the minimal side (e.g. `full`).  Users for whom a minimal install is preferred are, in my experience, the more expert users who know precisely what they need and are looking to streamline their deploys.  Such experienced users are much better at discovering configuration options and adjusting them.  More novice users almost always accept default options, and benefit the most when those defaults are very "batteries included".

Reply:
Aside from `rlang`, if `jsonlite` is not installed, an infinite number of error messages are displayed. (REditorSupport/vscode-R#1204)
So I thought it was necessary to install at least `jsonlite`.
The `jsonlite` and `rlang` installations are fast, so we don't have to wait minutes for the source installation like with the other options.